docs/devel/flaky-tests.md

```diff
@@ -194,6 +194,41 @@ kubectl delete replicationcontroller flakecontroller
 
 If you do a final check for flakes with `docker ps -a`, ignore tasks that exited -1, since that's what happens when you stop the replication controller.
 
+
+# Reproducing integration test flakes
```

Comment:
Title seems misleading. Contents talks about starting multiple etcd servers. It doesn't explain how this ties into the title.